scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -128,6 +128,11 @@ def test_constructor(self):
         assert_array_equal(s4.center, center)
         assert_equal(s4.radius, radius)
 
+        # Test a non-sequence/-ndarray based array-like
+        s5 = SphericalVoronoi(memoryview(self.points))
```

Comment:
The fix looks good to me, thanks.

Can you briefly explain to me if it is safe to ignore the `mypy` CI error that shows up for the new test?

```python
scipy/spatial/tests/test_spherical_voronoi.py:132: error: Argument 1 to "memoryview" has incompatible type "ndarray[Any, dtype[Any]]"; expected "Union[bytes, bytearray, memoryview, array[Any], mmap]"  [arg-type]
```

Reply:
Ah, this definitely needs a `type: ignore` comment.

In any case, this is an example infamous buffer protocol issue, more specifically our inability to properly type it, 
as it is defined in C (xref https://github.com/python/typing/issues/593). As a compromise, typeshed currently uses an [inexhaustive union](https://github.com/python/typeshed/blob/bc2ec748f68dcb516a9ed83c6b8a494f31cf3698/stdlib/_typeshed/__init__.pyi#L151-L152)
 for representing buffer protocol implementations. 

Unfortunately, this is one of the cases where its inexhaustiveness really shows.